sale_report_delivered_volume/models/sale_report.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2022 Tecnativa - Carlos Roca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class SaleReport(models.Model):
+    _inherit = "sale.report"
+
+    volume_delivered = fields.Float(digits="Volume")
+
+    def _query(self, with_clause="", fields=None, groupby="", from_clause=""):
+        if fields is None:
+            fields = {}
+        fields[
+            "volume_delivered"
+        ] = ", sum(p.volume * l.qty_delivered / u.factor * u2.factor) as volume_delivered"
```

Comment:
Why extend this method instead of extend _select 
https://github.com/OCA/sale-reporting/blob/caa9682f871866ba042c503b756c700c5cd77961/sale_report_delivered/reports/sale_report.py#L62
or _sub_select
https://github.com/OCA/sale-reporting/blob/caa9682f871866ba042c503b756c700c5cd77961/sale_report_delivered/reports/sale_report.py#L103

Reply:
We are extending sale, not `sale_report_delivered`

Reply:
Sorry... The module name...